backend/src/main/java/com/htecgroup/skynest/controller/UserController.java

```diff
@@ -13,16 +13,25 @@
 public class UserController {
 
   @Autowired private UserService userService;
-  @Autowired private ModelMapper modelMapper;
 
   @PostMapping("/register")
   public UserResponse registerUser(@RequestBody UserRegisterRequest userRegisterRequest) {
 
-    UserDto userDto = userService.registerUser(modelMapper.map(userRegisterRequest, UserDto.class));
+    ModelMapper modelMapper = new ModelMapper();
```

Comment:
Would it be better and possible if the ModelMapper is part of sprint and inject it here? Pls check if that is done in some other PRs